client/card-readers/settings/file-upload.tsx

```diff
@@ -24,16 +37,18 @@ const BrandingFileUpload = ( props ) => {
 		purpose,
 		fileID,
 		updateFileID,
-	} = props;
+	}: CardReaderFileUploadProps = props;
```

Comment:
Can we remove `: CardReaderFileUploadProps`? Since we are already defining the type of props in `const BrandingFileUpload: React.FunctionComponent< CardReaderFileUploadProps >` I was wondering if TS compiler needs it?